dapps/onboarding.vue/src/i18n/de.json

```
@@ -93,27 +73,46 @@ export default {
       "get-mnemonic-desc": "Sichern Sie Ihren Wiederherstellungsschlüssel.",
       "get-mnemonic-desc-long": "Mit Hilfe dieses <b>Wiederherstellungsschlüssels</b> können Sie Ihre Identität auf <b>jedem beliebigen Gerät</b> wiederherstellen und auf Ihre Daten zugreifen. Bitte <b>sichern Sie sich diese 12 Wörter in einer möglichst sicheren Weise</b> (möglichst offline, z.B. auf einem Blatt Papier im Safe). <br><br> Wenn Sie diese 12 Wörter verlieren, <b>verlieren Sie den Zugang zu Ihren Geschäftspartnern, Verträgen und Profilinformationen.</b>",
       "headings": {
-        "header-0" : "Ihr Tor zur Digitalisierung",
         "desc-0": "Digitale Identitäten allein bilden noch kein Netzwerk. Es wird auch eine neutrale operative Plattform, auf der Transaktionen zwischen Dingen, Organisationen und Menschen sicher und zuverlässig abgewickelt werden können, benötigt.",
-        "header-1": "Transparenz",
         "desc-1": "Die Blockchain-Technologie ermöglicht die Transparenz von Transaktionen durch die Sicherstellung einer nicht personalisierten Transparenz. Jede Transaktion kann von jedem Mitglied der Blockchain eingesehen werden. Ihre Privatsphäre wird geschützt, indem entschlüsselte Adressen anstelle von einfachen Namen verwendet werden.",
+        "header-0": "Ihr Tor zur Digitalisierung",
```

Tor zur Digitalisierung klingt mega holprig, vielleicht lieber sowas wie das: ? 
```suggestion
        "header-0": "Ihr Digitalisierungsgateway",
```